.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  release:
+    types: [ created ]
```

Comment:
what does `created` mean?

Reply:
This is saying run the workflow when a release is created